scripts/nuget.autopkg

```diff
@@ -1,3 +1,11 @@
+configurations {
+      	// This node contains custom pivot information.
+      	Toolset 
+      	{
+           	key: "PlatformToolset";
+           	choices: { v140, v120, v110, v100 };
```

Comment:
@zeux I am also quite unfamiliar with NuGet. I had found this on StackOverflow as a workaround for the fact that the CoApp NuGet autopkg doesn't have the configuration included for VS 2015 yet. It probably will work either way.